http/oidc/src/main/java/org/wildfly/security/http/oidc/AuthenticationError.java

```diff
@@ -36,7 +36,9 @@ public enum Reason {
         INVALID_TOKEN,
         STALE_TOKEN,
         NO_AUTHORIZATION_HEADER,
-        NO_QUERY_PARAMETER_ACCESS_TOKEN
+        NO_QUERY_PARAMETER_ACCESS_TOKEN,
+        NO_SESSION_ID,
+        METHOD_NOT_ALLOWED
```

Comment:
I think NO_SESSION_ID and METHOD_NOT_ALLOWED can be removed, I don't see them being used.

Reply:
done